packages/esm-patient-orders-app/src/components/orders-details-table.component.tsx

```diff
@@ -134,13 +137,22 @@ const OrderDetailsTable: React.FC<OrderDetailsProps> = ({ patientUuid, showAddBu
           launchAddDrugOrder({ order: buildMedicationOrder(orderItem, 'REVISE') });
           break;
         case 'testorder':
-          launchModifyLabOrder({ order: buildLabOrder(orderItem, 'REVISE') });
+          launchModifyLabOrder({
+            order: buildLabOrder(orderItem, 'REVISE'),
+            orderTypeUuid: orderItem.orderType.uuid,
```

Comment:
I don't quite understand why we need this change for `launchModifyLabOrder`?

Reply:
I forgot to add this in the PR description, I'll add it now, but the order basket item grouping before was either `medications` or `labs`, but except Drug orders (`medications`), the grouping for all other orders is actually the `orderType` of the order.